pkg/controllers/provisioningv2/cluster/scheduling_customization.go

```diff
@@ -11,11 +11,6 @@ import (
 	"github.com/rancher/rancher/pkg/settings"
 )
 
-// updateV1SchedulingCustomization looks for the provisioning.cattle.io/enable-scheduling-customization annotation on the v1.Cluster
-// and if found populates the spec.ClusterAgentDeploymentCustomization.SchedulingCustomization field with the default values set in the
-// global settings. If the cluster-agent-scheduling-customization feature is disabled, the cluster will be returned unchanged.
-// The provisioning.cattle.io/enable-scheduling-customization annotation can be set to 'true' or 'false', which will add or remove
-// the scheduling customization field as needed.
 func (h *handler) updateV1SchedulingCustomization(_ string, cluster *v1.Cluster) (*v1.Cluster, error) {
```

Comment:
Where did the comment go?

Reply:
Basically the comments moved into the more specific `updateV1AgentSchedulingCustomization` and `updateV1FleetAgentSchedulingCustomization` respectively. For `updateV3SchedulingCustomization` they moved in to the more specific `updateV3AgentSchedulingCustomization` and `updateV3FleetAgentSchedulingCustomization`. But I re-added a more generic version to `updateV1SchedulingCustomization` and `updateV3SchedulingCustomization`.